app/javascript/packs/usersEventList.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from "react";
+import ReactDOM from "react-dom";
+import axios from "axios";
+
+class UsersEventList extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      value: "",
+      addedUsers: [],
+      users: []
+    };
+  }
+
+  componentWillMount() {
+    axios
+      .get("/api/v1/users.json", {}, { "Content-Type": "application/json" })
+      .then(res => {
+        this.setState({
+          users: res.data.data
+        });
+      });
+  }
+
+  handleChange = event => {
+    this.setState({ value: event.target.value });
+  };
+
+  handleSubmit = event => {
+    if (this.state.addedUsers.length < 4) {
+      this.setState({
+        addedUsers: [
+          ...this.state.addedUsers,
+          this.state.users.find(user => user.nick === this.state.value)
+        ]
+      });
+      alert("new user was added");
```

Comment:
Get rid of this alert. Information about added player is already accessible in the form of his nick on the list of added users.